### ISSUE TITLE
Full Screen View / Distraction Free mode

### ISSUE THREAD

**Issue by plug-n-play** [KhalisFoundation/sttm-web#692]
It would be nice to have a Full-Screen toggle, preferably on the bottom right as under

![image](https://user-images.githubusercontent.com/6245927/58541347-eee01680-8218-11e9-8208-ddc3e82ee914.png)

When clicked, user enters full screen mode. E.g. :
![image](https://user-images.githubusercontent.com/6245927/58541524-5302da80-8219-11e9-9f1b-5996418443b4.png)

Full Screen view can have a cross-icon on top right to return to original view

Additionally, keyboard shortcuts would be great. Like `f/F` to enter full screen mode. `Esc` or pressing `f/F` returns back to normal mode

**Comment by tarunsingh5** [comment]
love this idea!

**Comment by AkalUstat** [comment]
@saintsoldierx i can also attempt this after #439

**Comment by Gauravjeetsingh** [comment]
Merged to prod. Closing this out.
